### ISSUE TITLE
Support for named import pattern in import/named

### ISSUE THREAD

**Issue by maknahar** [benmosher/eslint-plugin-import#1583]
Given:

```js
// ./foo.js
export const foo = "I'm so foo"
```

The following Es6 syntax is considered invalid:

```js
// ./bar.js
import { foo as bar } from './foo'
```

Error: 
>ESLint: foo not found in './foo'(import/named)

However, the following is considered valid

```js
// ./bar.js
import * as bar from './foo'
```

is it done intentionally? And if so, do we have an option to disable errors? 

My configuration:
```
{
    "env": {
        "browser": true,
        "jest": true
    },
    "parserOptions": {
        "sourceType": "module",
        "ecmaFeatures": {
            "jsx": true
        }
    },
    "extends": [
        "airbnb",
        "plugin:prettier/recommended"
    ],
    "plugins": [
        "react"
    ],
    "rules": {
        "class-methods-use-this": 0,
        "import/no-named-as-default": 0,
        "react/jsx-filename-extension": [
            "error",
            {
                "extensions": [
                    ".js",
                    ".jsx"
                ]
            }
        ]
    }
}
```

Eslint version: v6.8.0

**Comment by ljharb** [comment]
Duplicate of #1584.

**Comment by maknahar** [comment]
@ljharb Could you please let me know how this is duplicate of #1584?

#1584 Talks about the issue where we have an index.js file which combines few imports from other modules and export them together with `module.exports = {}` syntax.

This issue is that eslint is giving an error if you use es6 syntax and rename it like `import { foo as bar } from './foo'`. 

Could you please explain to me how both of these issues are similar?

**Comment by ljharb** [comment]
You’re right; i misread it earlier.

**Comment by gtsop** [comment]
What's the status with this?
